src/cli/cli_br.cpp

```diff
@@ -435,6 +435,45 @@ template <> otError Br::Process<Cmd("prefixtable")>(Arg aArgs[])
     return error;
 }
 
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE
+template <> otError Br::Process<Cmd("pd")>(Arg aArgs[])
+{
+    otError error = OT_ERROR_NONE;
+
+    VerifyOrExit(aArgs[0].IsEmpty(), error = OT_ERROR_INVALID_ARGS);
+
+    /**
+     * @cli br pd state
+     * @code
+     * br pd state
+     * running
+     * Done
+     * @endcode
+     * @par api_copy
+     * #otBorderRoutingDhcp6PdGetState
+     */
+    if (aArgs[0] == "state")
+    {
+        static const char *const kDhcpv6PdState[] = {"disabled", "stopped", "running"};
+        static_assert(0 == OT_BORDER_ROUTING_DHCP6_PD_STATE_DISABLED,
+                      "The value of OT_BORDER_ROUTING_DHCP6_PD_STATE_DISABLED is not expected!");
+        static_assert(1 == OT_BORDER_ROUTING_DHCP6_PD_STATE_STOPPED,
+                      "The value of OT_BORDER_ROUTING_DHCP6_PD_STATE_STOPPED is not expected!");
+        static_assert(2 == OT_BORDER_ROUTING_DHCP6_PD_STATE_RUNNING,
+                      "The value of OT_BORDER_ROUTING_DHCP6_PD_STATE_RUNNING is not expected!");
+
+        OutputLine("%s", Stringify(otBorderRoutingDhcp6PdGetState(GetInstancePtr()), kDhcpv6PdState));
+    }
+    else
```

Comment:
Should we add "br pd enable/disable" as well?

Reply:
we don't have API yet, plan to add enable/disable next time

Reply:
We have `otBorderRoutingDhcp6PdSetEnabled()`?

Reply:
ohh cool, I will add it later!